### PR TITLE
Update ci-theme.css

### PR DESCRIPTION
**Pull request by demyr17**
Turned 2 vertical boxes into horizontal view. When we first prepared there were 3 versions of CI there : 4 - 3 and 2.. But then, only with 2 vertical boxes the page look was not meaningful without the third box.

### CODE REVIEW

**Comment by MGatner** [comment]
Before:
before

After:
after

File: public/assets/css/ci-theme.css [hunk 1]
```diff
@@ -1288,13 +1288,12 @@ section#content-outer-discover{
 
 
 .ci-version-boxes{
-
 	float:left;
-	width:31%;
-	min-height:700px; /*240px*/
+	width:100%;/*31%*/
```
Comment (MGatner) [inline public/assets/css/ci-theme.css:1292]: No need to leave comments in place - `git` will track the whole history so cleaner just to remove things.
```suggestion
	width:100%;
```

File: public/assets/css/ci-theme.css [hunk 2]
```diff
@@ -1288,13 +1288,12 @@ section#content-outer-discover{
...
-
 	float:left;
-	width:31%;
-	min-height:700px; /*240px*/
+	width:100%;/*31%*/
+	min-height:300px;
 	margin-top:30px;
 	margin-bottom:45px;
-	margin-right:2%;
+	/*margin-right:2%;*/
```
Comment (MGatner) [inline public/assets/css/ci-theme.css:1296]:
```suggestion
```

File: public/assets/css/ci-theme.css [hunk 3]
```diff
@@ -1319,11 +1318,11 @@ section#content-outer-discover{
...
 
 .cv-boxes-version{
-		.float:left;
+		float:left;
 		width:180px;
 		height:180px;
-		margin:30px auto;
-		.margin-left:10px;
+		margin:30px; /*30px auto;*/
+		/*margin-left:10px;*/
```
Comment (MGatner) [inline public/assets/css/ci-theme.css:1325]:
```suggestion
		margin:30px;
```

File: public/assets/css/ci-theme.css [hunk 4]
```diff
@@ -1357,11 +1356,11 @@ span.version-no{
 
 .cv-boxes-buttons-area	{
 	float:left;
-	width:80%;
+	width:70%;/*80%*/
```
Comment (MGatner) [inline public/assets/css/ci-theme.css:1359]:
```suggestion
	width:70%;
```

File: public/assets/css/ci-theme.css [hunk 5]
```diff
@@ -1357,11 +1356,11 @@ span.version-no{
 
 .cv-boxes-buttons-area	{
 	float:left;
-	width:80%;
+	width:70%;/*80%*/
 	height:100px;
 	position:absolute;
 	bottom:0;
-	left:10%;
+	left:22%; /*10%*/
```
Comment (MGatner) [inline public/assets/css/ci-theme.css:1363]:
```suggestion
	left:22%;
```

**Review by MGatner — CHANGES_REQUESTED** [review]
Looks better! Thanks for this. No need to leave comments on the old versions of things as that will all be available in the revision history. I made suggestions to remove these - if you click the "Files" tab you can use the "Add suggestion to batch" buttons to add all of them to a new commit and GitHub will update the PR for you.

**Comment by demyr17** [comment]
> 
> 
> Looks better! Thanks for this. No need to leave comments on the old versions of things as that will all be available in the revision history. I made suggestions to remove these - if you click the "Files" tab you can use the "Add suggestion to batch" buttons to add all of them to a new commit and GitHub will update the PR for you.

Is there anything I should do now? or was your warning for the future?

**Comment by MGatner** [comment]
You need to update the PR. You can use the process I outlined above, or you can submit a new PR without the comments if that is easier. I recommend learning how to add commits from GitHub suggestions - it saves a lot of time for small changes.

**Review by MGatner — APPROVED** [review]
Thanks Demyr! Looks good.

**Pull request merged**